third_party/txt/src/txt/paragraph.cc

```diff
@@ -223,14 +223,11 @@ void Paragraph::Layout(double width, bool force) {
   lines_ = 0;
   line_widths_ = std::vector<double>();
   line_heights_ = std::vector<double>();
-  line_heights_.push_back(0);
   records_ = std::vector<PaintRecord>();
 
   // Set padding elements to have a minimum point.
-  glyph_position_x_ = std::vector<std::vector<double>>();
-  glyph_position_x_.push_back(std::vector<double>());
-  std::vector<double> glyph_single_line_position_x;
-  glyph_single_line_position_x.push_back(0);
+  glyph_position_x_ = std::vector<std::vector<GlyphPosition>>();
+  std::vector<GlyphPosition> glyph_single_line_position_x;
```

Comment:
Update (or eliminate) the comment above now that we don't need to push the initial zero position start.